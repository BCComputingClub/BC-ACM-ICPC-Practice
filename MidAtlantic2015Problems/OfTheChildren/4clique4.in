6
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
-1 -1 -1
11
2
3
3
4
0
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
5 6 1
9 10 100
6 7 2
6 8 3
6 9 4
7 8 5
7 9 6
8 9 1
-1 -1 -1
16
2
3
3
4
0
2
3
3
4
0
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
5 6 1
9 10 100
6 7 2
6 8 3
6 9 4
7 8 5
7 9 6
8 9 1
10 11 1
14 15 100
11 12 3
11 13 4
11 14 5
12 13 6
12 14 1
13 14 2
-1 -1 -1
21
2
3
3
4
0
2
3
3
4
0
2
3
3
4
0
2
3
3
4
0 1 1
4 5 100
1 2 1
1 3 2
1 4 3
2 3 4
2 4 5
3 4 6
5 6 1
9 10 100
6 7 2
6 8 3
6 9 4
7 8 5
7 9 6
8 9 1
10 11 1
14 15 100
11 12 3
11 13 4
11 14 5
12 13 6
12 14 1
13 14 2
15 16 1
19 20 100
16 17 4
16 18 5
16 19 6
17 18 1
17 19 2
18 19 3
-1 -1 -1
0
