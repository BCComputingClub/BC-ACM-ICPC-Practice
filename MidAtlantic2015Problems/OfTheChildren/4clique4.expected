98
194
290
387
